diff --git a/ckan/ckan/lib/munge.py b/ckan/ckan/lib/munge.py
index fa2089d54..540edcadf 100644
--- a/ckan/ckan/lib/munge.py
+++ b/ckan/ckan/lib/munge.py
@@ -16,7 +16,7 @@ from ckan.lib.io import decode_path
 MAX_FILENAME_EXTENSION_LENGTH = 21
 
 # Maximum total length of a filename (including extension)
-MAX_FILENAME_TOTAL_LENGTH = 100
+MAX_FILENAME_TOTAL_LENGTH = 10100
 
 # Minimum total length of a filename (including extension)
 MIN_FILENAME_TOTAL_LENGTH = 3

diff --git a/ckan/ckan/lib/helpers.py b/ckan/ckan/lib/helpers.py
index 24249cabd..c0d96a34a 100644
--- a/ckan/ckan/lib/helpers.py
+++ b/ckan/ckan/lib/helpers.py
@@ -2593,7 +2593,7 @@ def get_organization(org: Optional[str] = None,
         return {}
     try:
         return logic.get_action('organization_show')(
-            {}, {'id': org, 'include_datasets': include_datasets})
+            {'ignore_auth': True}, {'id': org, 'include_datasets': include_datasets})
     except (logic.NotFound, logic.ValidationError, logic.NotAuthorized):
         return {}
 
